Cargo.toml

```diff
@@ -12,20 +12,23 @@ readme = "README.md"
 exclude = ["examp", "fixtures"]
 default-run = "cargo-sort"
 
+[features]
+fuzz = ["afl"]
+
 [dependencies]
-afl = { version = "0.10.0", optional = true }
-clap = "2.33.3"
-glob = "0.3.0"
+afl = { version = "0.10", optional = true }
+clap = "2.33"
+glob = "0.3"
 termcolor = "1.1"
 
 # toml_edit's dependencies
-chrono = "0.4.10"
-combine = "3.8.1"
-linked-hash-map = "0.5.2"
+chrono = "0.4"
+combine = "3.8"
+linked-hash-map = "0.5"
 
 [dev-dependencies]
-pretty_assertions = "0.7.1"
-serde_json = "1.0.44"
+pretty_assertions = "0.7"
+serde_json = "1"
```

Comment:
What's the reason for removing patch versions (just curious)? And why is this not `1.0`? :)

Reply:
Well if it follows semver I can get all the update up to 1.9 without breaking right?

Reply:
It can with the previous version bound too. All you're doing here is lowering the minimum version. `1`, `1.0` and `1.0.0` are exactly equal version bounds. Version requirements are implicitly caret req's ([docs](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements)), if you wanted exactly `1.0.44` that would be written as `serde_json = "=1.0.44"`.

Reply:
Ahh cool good to know!